LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js

```diff
@@ -33,11 +33,11 @@ async function dispatchMouseActions(actions, pointerType)
         switch (action.type) {
         case "pointerMove":
             const origin = { x: 0, y: 0 };
-            if (action.origin instanceof Element) {
+            if (action.origin?.ownerDocument?.defaultView && action.origin instanceof action.origin.ownerDocument.defaultView.Element) {
```

Comment:
```suggestion
            if (action.origin instanceof action.origin?.ownerDocument?.defaultView?.Element) {
```
Doesn't this work?

Reply:
I had this originally, but It did not work because in the case where it is null, a JS exception is thrown because the right hand side of `instanceof` must be a non null object

Reply:
Maybe store the window object in a variable to avoid duplicatation?

Reply:
will do; I also need to figure out what to do about the newly failing tests 😔